src/main/java/com/forgeessentials/api/UserIdent.java

```diff
@@ -22,9 +31,32 @@
 
 import cpw.mods.fml.common.eventhandler.Event;
 
+import javax.net.ssl.HttpsURLConnection;
+
 public class UserIdent
 {
 
+    public static UUID hexStringToUUID(String s)
+    {
+        if (s.length() != 32)
+            throw new IllegalArgumentException();
+        byte[] data = new byte[32];
+        for (int i = 0; i < 32; i++)
+        {
+            data[i] = (byte) s.charAt(i);
+        }
+        return UUID.nameUUIDFromBytes(data);
+    }
+
+    public static UUID stringToUUID(String s)
+    {
+        if (s.length() == 32)
+            return hexStringToUUID(s);
```

Comment:
Blame Mojang and their api. They output their uuids without any dashes and the normal system can't handle them, so I have to handle them specially.